### ISSUE TITLE
Remove "incubating"

### ISSUE THREAD

**Issue by UncleGedd** [apache/flagon#27]
Now that we've graduated, we should update the website and remove references to us being in incubation. Noting that the web address itself needs an update: http://flagon.incubator.apache.org/

**Comment by poorejc** [comment]
@UncleGedd yes!! This is definitely an issue in our post-TLP steps that we absolutely need to address. 

How the site works--see here: https://github.com/apache/flagon/tree/asf-site/site

BLUF--update source at MASTER, rebuild and copy files to /contents, then merge to ASF-SITE branch. Automation for publication is managed through the asf.yaml files in asf-site branch.

The site could use a major overhaul in the build (its so old). We can limp by updating files. I can try and help this weekend in updating the .md files which build into html.

**Comment by poorejc** [comment]
Progress tracker:
site/_docs
- [x] UserALE.js
- [ ] Distill
- [ ] Tap
- [ ] Update Apache Logo

**Comment by brucearctor** [comment]
<img width="1207" alt="Screenshot 2023-06-10 at 2 40 09 PM" src="https://github.com/apache/flagon/assets/5032356/16971eb1-ce8d-46a5-9684-a3c2be341107">


Notice incubating also appears in the link on the GH repos.
